HousingSearchApi/V1/Infrastructure/Factories/AssetQueryGenerator.cs

```diff
@@ -113,7 +113,7 @@ public QueryContainer Create<TRequest>(TRequest request, QueryContainerDescripto
                         .WithMultipleFilterQuery(assetListAllRequest.ParentAssetId, new List<string> { "rootAsset" })
                         .WithMultipleFilterQuery(assetListAllRequest.IsActive, new List<string> { "isActive" })
                         .WithMultipleFilterQuery(assetListAllRequest.ContractIsApproved, new List<string> { "assetContract.isApproved" })
-                        .WithMultipleFilterQuery(assetListAllRequest.ContractApprovalStatus.ToString(), new List<string> { "assetContract.approvalStatus" })
+                        .WithMultipleFilterQuery(assetListAllRequest.ContractApprovalStatus, new List<string> { "assetContract.approvalStatus" })
```

Comment:
If this is still an enum and you want that `.ToString()`, I think you can just add a `?`:

```cs
assetListAllRequest.ContractApprovalStatus?.ToString()
```

Reply:
yeah, but at that point it's a string.